Add mission CRF_S&D124_SemperViva to the repository

FluffSquirrel opened the pull request:
Mission Pull Request Template

This Pull Request will:

Add the mission Semper Viva made by fluff_ to the repository.

**Mission Creation Checklist:**
- [x] Mission File
- [x] Ingame Play Area Markers
- [x] Ingame Briefings to all players
- [x] Side-based briefings
- [x] Objective Entities
- [x] Mission .conf file

**Faction(s):**
BLUFOR - Attacking
- [ ] US
- [ ] UK
- [ ] RHS_USAF 
- [x] CDF

OPFOR - Defending
- [ ] USSR
- [ ] RHS_AFRF
- [x] ChDKZ

INDFOR - NA
- [ ] FIA
- [ ] RHS_ION
- [ ] TALIBAN

**Objectives/Win Conditions:**

CDF destroy the two bombsites inside of Pustoshka

**Terrain:**

Chernarus

**Short mission description:**

CDF forces conduct a motorized assualt into the town of Pustoshka

**Slotting Ratio:**

3:2 blufor opfor

**Time limit:**

75m

**Mission Briefing Screenshot:**
![image](https://github.com/user-attachments/assets/741a9a11-5363-4b8f-b94a-050b592bcb6e)

**Design concept/thoughts:**

BTR balance is biggest concern. The Combat engineers get 12 mines which im sure could be useful, but probably wont go very far. I reckon the ChDKZ will have at least 2-3 infantry squads with AT though, and for each AT slot filled thats 3 HEAT rockets, and two frag rockets that might be able to harm the btr (not sure) Since 1 heat rocket is really all it takes for a crewkill or even a cookoff, ~~and adding the two PKMs from mmg that can do damage to crew in the less armored spots,~~<sub>  Harry showed me that the pkms dont damage BTRs anymore. I still think its alright, and if an an adjustment needs to be made I think MAT is an overkill, maybe a LAT team</sub> I think the 3 BTRs are balanced.

need a go ahead on civ vics in the ao, their inventories are emptied so no bandages in them and you cant enter them, minus you can sit in the back of the civilian ural transport but i dont really see a problem there

**Other Notes:**

<Assets? Admin notes? Anything else?>
BLUFOR:
3 btr
3 ural
1 uaz
2 med vics
1 recon team

opfor
1 med vic
1 combat engineer team w mines
1 mmg team

Players need to know OPFOR leadership has single channel radios, and BLUFOR uses RTO structure. BLUFOR PSG, 1SG, MO are acting RTOs for command elements. No GI radios on either side.

RandAlThor1 commented:
**QA Feedback**
- you need to specifiy AT Mines, also in both sides briefing, I would also include it in both sides briefings
-  you need to give a better descriptor of what the actual bombs are, no one is going to read the wall of text in the lore section
- Recon cannot be apart of briefing if they spawn away from everyone, nor are forward observers used here, FOs are for indirect weapons like mortars. a recon team would be something like a SL/RTO/Rifleman here, tho I also fail to see the value they add to CDF, if you want to use them spawn them with everyone else and hope that BLUFOR uses them appropriately.
- 75 minutes is a long time for this mission, bring it down to 40/45
- Medical is a COY asset here don't need 5 medics per platoon, nor will they ever be slotted.
![image](https://github.com/user-attachments/assets/3647e4aa-3309-457f-87fe-4eac54fc999c)

FluffSquirrel commented:
changes:
added bombsite description to situation,
 removed recon team
  removed extra medical team, 
  mission time 50m, 
  more specificity towards anti tank mines
